setup.py

```diff
@@ -66,7 +65,6 @@ def get_version():
         "Natural Language :: English",
         "Operating System :: OS Independent",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
```

Comment:
the issue with python 3.6 is that it doesn't support `from __future__ import annotations` (which makes annotations lazily evaluated)

Reply:
@lopuhin do you foresee anything breaking in some of our stuff if we drop 3.6? I'm inclined to drop it here since it reached its EOL for almost a year now. 

Moreover, I think it'd be great to formally support 3.10 and 3.11 while we're at it.

Reply:
@BurnzZ I'm +1 to drop python 3.6 since it's EOL, and support for 3.10 and 3.11 would be great too :)